RecoEgamma/EgammaElectronProducers/python/ecalDrivenElectronSeedsParameters_cff.py

```diff
@@ -81,3 +81,5 @@
     
 )
 
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+pp_on_AA_2018.toModify(ecalDrivenElectronSeedsParameters, SCEtCut = cms.double(15.0))
```

Comment:
please drop the types in all modifications.
This is a safer use of syntax against typos and also helps with possible future parameter name changes and migrations